diagnostic.md

```diff
@@ -36,19 +36,19 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+relative, because it doesnt start from root "~"
```

Comment:
'~' is shorthand for the home directory, but a path that begins with '/' is the definition of an absolute path. Try this - do 'cd ~' in command line and then do 'ls'. Then try 'cd /' and then 'ls'. You'll see that those are 2 different places. The root directory != home directory.